### ISSUE TITLE
[FEATURE REQUEST] Add support for pino-http's customAttributeKeys.err option

### ISSUE THREAD

**Issue by dougludlow** [iamolegga/nestjs-pino#1662]
**Is your feature request related to a problem? Please describe.**
`pino` allows setting the [`errorKey`](https://github.com/pinojs/pino/blob/master/docs/api.md#errorkey-string), the "string key for the 'error' in the JSON object" and `pino-http` allows customizing the `err` log object attribute via [`opts.customAttributeKeys.key`](https://github.com/pinojs/pino-http#pinohttpopts-stream), but `nestjs-pino` is hardcoded to use `err` in the `PinoLogger`: https://github.com/iamolegga/nestjs-pino/blob/3.4.0/src/PinoLogger.ts#L138

We use Datadog and it requires that errors are in the `error` attribute of the log object, effectively ignoring `err`.

**Describe the solution you'd like**
Consider using the `pinoHttp.customAttributeKeys.err` option when passing the error into the merge object field.

Happy to put up a pull request if you'd like.

**Describe alternatives you've considered**
I can override the `call` method of the `PinoLogger` in a derived class to account for the `pinoHttp.customAttributeKeys.key` option, this is my current solution, but I've love to see this implemented natively in `nestjs-pino`.

Something like this: 
```
  protected readonly errorKey = 'err';

  constructor(
    @Inject(PARAMS_PROVIDER_TOKEN) { pinoHttp, renameContext }: Params,
  ) {
    // ...

    if (
      typeof pinoHttp === 'object' &&
      'customAttributeKeys' in pinoHttp &&
      typeof pinoHttp.customAttributeKeys !== 'undefined'
    ) {
      this.errorKey = pinoHttp.customAttributeKeys.err ?? 'err';
    }
  }
```

```
  protected call(method: pino.Level, ...args: Parameters<LoggerFn>) {
    //...
        if (firstArg instanceof Error) {
          args = [
            Object.assign(
              { [this.contextName]: this.context },
              { [this.errorKey]: firstArg },
            ),
            ...args.slice(1),
          ];
        }
    //...
  }
```

**Comment by iamolegga** [comment]
LGTM, fill free to open a PR for this 👍

**Comment by dougludlow** [comment]
Ok, opened up a PR. Feel free to take a look when you have a moment.

**Comment by iamolegga** [comment]
published in 3.5.0
